src/main/java/org/cactoos/scalar/Folded.java

```diff
@@ -23,94 +23,55 @@
  */
 package org.cactoos.scalar;
 
-import java.util.Iterator;
-import java.util.NoSuchElementException;
 import org.cactoos.BiFunc;
 import org.cactoos.Scalar;
 
 /**
- * Folds iterable via BiFunc.
+ * Iterable, which elements are "folded" through the func.
  *
- * <pre>
- * new Folded<>(
- *     (first, last) -> first + last,
- *     new IterableOf<>(() -> 1L, () -> 2L, () -> 3L, () -> 4L)
- * ).value() // returns 10L
- * </pre>
- *
- * <p>Here is how you can use it to
- * find one of items according to the specified {@link BiFunc}:</p>
- *
- * <pre>
- *  final String apple = new Folded&lt;&gt;(
- *          (first, last) -&gt; first,
- *          new IterableOf&lt;Scalar&lt;String&gt;&gt;(
- *              () -&gt; "Apple",
- *              () -&gt; "Banana",
- *              () -&gt; "Orange"
- *          )
- *      ).value();
- *  final String orange = new Folded&lt;&gt;(
- *          (first, last) -&gt; last,
- *          new IterableOf&lt;Scalar&lt;String&gt;&gt;(
- *              () -&gt; "Apple",
- *              () -&gt; "Banana",
- *              () -&gt; "Orange"
- *          )
- *      ).value();
- * </pre>
- *
- * <p>There is no thread-safety guarantee.
- *
- * <p>This class implements {@link Scalar}, which throws a checked
- * {@link Exception}. This may not be convenient in many cases. To make
- * it more convenient and get rid of the checked exception you can
- * use {@link UncheckedScalar} or {@link IoCheckedScalar} decorators.</p>
- *
- * @author Alexander Dyadyushenko (gookven@gmail.com)
- * @author Eduard Balovnev (bedward70@mail.ru)
+ * @author Yegor Bugayenko (yegor256@gmail.com)
  * @version $Id$
- * @param <T> Scalar type
- * @since 0.29
```

Comment:
@VsSekorin I'm not sure about the changing `@since` version. You transfer authors with code which they wrote - it's okay, but a class with this name appeared in a different version of the lib. But keeping old version looks incorrectly for me too because that functionality appeared in a different version. So I think the best option is a putting current version `29.3` in both classes which shows to users when this API (class name + functionality) appeared.